.github/workflows/tests_primary.yml

```diff
@@ -71,3 +71,20 @@ jobs:
       if: matrix.os != 'ubuntu-latest'
     - run: xvfb-run npm run ttest
       if: matrix.os == 'ubuntu-latest'
+
+  test_runner_vrt:
+    name: Test Runner Visual Regression Testing
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 12
+    - run: npm i -g npm@7
+    - run: npm ci
+      env:
+        DEBUG: pw:install
+    - run: npm run build
+    - run: npx playwright install --with-deps
```

Comment:
Do we need browsers and deps here since it's in docker iiuc?

Reply:
good point, probably not